src/builder/rollup.ts

```diff
@@ -178,14 +183,11 @@ export async function rollupBuild(ctx: BuildContext) {
 
   // Types
   if (ctx.options.declaration) {
-    rollupOptions.plugins = rollupOptions.plugins || [];
-    // TODO: Use fresh rollup options
-    const shebangPlugin: any = rollupOptions.plugins.find(
-      (p) => p && p.name === "unbuild-shebang",
-    );
-    shebangPlugin._options.preserve = false;
-
-    rollupOptions.plugins.push(dts(ctx.options.rollup.dts));
+    rollupOptions.plugins = [
+      rollupOptions.plugins,
```

Comment:
```suggestion
      ...rollupOptions.plugins,
```
Shouldn't this spread the original plugins list?